### ISSUE TITLE
Regression: "jar coordinate ... already loaded"  has returned.

### ISSUE THREAD

**Issue by jordansissel** [elastic/logstash#6781]
previous issue: https://github.com/elastic/logstash/issues/5159

```
--- jar coordinate com.fasterxml.jackson.core:jackson-databind already loaded with version 2.7.4 - omit version 2.7.3
--- jar coordinate com.fasterxml.jackson.core:jackson-annotations already loaded with version 2.7.0 - omit version 2.7.3
--- jar coordinate com.fasterxml.jackson.core:jackson-core already loaded with version 2.7.4 - omit version 2.7.3
--- jar coordinate com.fasterxml.jackson.core:jackson-databind already loaded with version 2.7.4 - omit version 2.7.3
--- jar coordinate com.fasterxml.jackson.core:jackson-core already loaded with version 2.7.4 - omit version 2.7.3
```

This occurs on master for at least 568666c7778a59c4c2b19c35b6dd79025f70b4ad. I haven't tested other branches.

**Comment by jordansissel** [comment]
I labeled this a blocker for reasons stated in [this comment](https://github.com/elastic/logstash/issues/5159#issuecomment-236274611) and copied here:

I'm marking this a blocker. The reasoning is we have bug reports from users asking about problems because of similar messages -- by similar, I mean messages that have the following properties:
1. the message is not actionable for the user
2. the message is confusing to the user
3. the message is not something caused by the user

When we have output to a user with the above properties, history has shown that this causes bad user experience. We can't ship with this kind of message.

**Comment by ph** [comment]
@jordansissel Is this occuring when you run `bin/logstash*` or in the tests?

**Comment by activeshadow** [comment]
@ph for what it's worth, I personally only see these messages when running tests.

**Comment by jakelandis** [comment]
This is happening due a mismatch in Jackson versions.  JRuby uses 2.7.3 via [jrjackson](https://github.com/guyboertje/jrjackson/blob/v0.4.2/jrjackson.gemspec#L24) and Java is targeted at [2.7.4](https://github.com/elastic/logstash/blob/v6.0.0-alpha2/logstash-core/build.gradle#L115). 

Not sure why this only shows up with tests, but some upcoming changes to move parts of the Jackson serialization to Java will surely at some-point cause these messages to show up in the logs. 

I don't think that downgrading from [2.7.4](https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.7.4) to 2.7.3 will have much if any impact.

I will log a separate issue to bump up the Jackson dependency across the board. (2.9 is the current)

**Comment by jakelandis** [comment]
Fixed in time for 6.0.0
